src/frontend/admin/src/components/templates/enrollments/buttons/EnrollmentActions.tsx

```diff
@@ -35,26 +35,29 @@ const messages = defineMessages({
     id: "components.templates.orders.buttons.enrollmentActionsButton.cantEnrollMessage",
     description: "Label for the can't enroll tooltip message",
     defaultMessage:
-      "It's impossible to enroll because the registration date has passed",
+      "It's impossible to enroll because the course run is not opened for enrollment.",
   },
   validateUnrollModalTitle: {
     id: "components.templates.orders.buttons.enrollmentActionsButton.validateUnrollModalTitle",
-    description: "Title for the validate unroll modal",
+    description:
+      "Text shown when clicking on the 'unroll' button to warn that it will not be possible to go back",
```

Comment:
I think previously I told you to use `unroll` term but recently I discovered that the right term is `unenroll` 😅 
```suggestion
      "Text shown when clicking on the 'unenroll' button to warn that it will not be possible to go back",
```